src/site/content/en/angular/route-preloading/index.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: 'Route preloading strategies in Angular'
+subhead: |
+  Preload routes ahead of time to speed up users' navigation!
+hero: route-preloading.jpg
```

Comment:
OK, sorry to be a pain, but I didn't realize this was a crystal ball until I read the alt text :)
Also, we might want to avoid stereotypes about the Roma. If you want to get at the idea of looking into the future, what about something like one of these?
- https://www.shutterstock.com/image-illustration/across-universe-traveling-space-time-travel-200832383
- https://www.shutterstock.com/image-illustration/colorful-3d-speed-tunnel-warp-234238399
- https://www.istockphoto.com/photo/time-machine-telegraph-going-to-the-future-gm838386376-136450705

Reply:
I picked this one because of the Guess.js logo https://github.com/guess-js/media. Not feeling strongly about it, we can go with anything which feels right.

Reply:
Fair enough. Letting it go :)